neural_compressor_ort/algorithms/layer_wise/core.py

```diff
@@ -0,0 +1,287 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2023 MIT HAN Lab
+# This source code is licensed under the MIT license
+#
+# Copyright (c) 2023 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+from copy import deepcopy
+from pathlib import Path
+from typing import Callable, List, Union
+
+import onnx
+import onnxruntime as ort
+import transformers
+
+from neural_compressor_ort.quantization.calibrate import CalibrationDataReader
```

Comment:
@mengniwang95 please check this.

Reply:
Hi @justinchuby , do you mean
```python
from neural_compressor_ort.quantization import calibrate
```
and then use CalibrationDataReader by calibrate.CalibrationDataReader ?
If not, please correct me

Reply:
Precisely. Thank you!

Reply:
Hi @justinchuby , we prefer to follow the style of ONNX repo rather than https://google.github.io/styleguide/pyguide.html#224-decision and will remove our coding_style doc, it it okay?